sql/src/test/java/io/crate/analyze/SelectStatementAnalyzerTest.java

```diff
@@ -210,9 +210,9 @@ public void testAllColumnNodes() throws Exception {
         QueriedRelation relation = analyze("select id, * from sys.nodes");
         List<String> outputNames = outputNames(relation);
         assertThat(outputNames.get(0), is("id"));
-        assertThat(outputNames.get(1), is("fs"));
-        assertThat(outputNames.size(), is(16));
-        assertThat(relation.querySpec().outputs().size(), is(16));
+        assertThat(outputNames.get(1), is("connections"));
```

Comment:
wouldn't it make more sense to assert all column names like done at `testAllColumnCluster`?

Reply:
yep, changed it